### ISSUE TITLE
Build does not work

### ISSUE THREAD

**Issue by elucify** [cvbarros/terraform-provider-teamcity#64]
HCL v2 is now released, so build instructions do not work.

Last build 4 months ago, but no indication from where builds could be downloaded.

Pity, seems like a nice package.

**Comment by cvbarros** [comment]
Hi @elucify 👋 
Would you mind elaborating on which errors you're getting regarding the build?

As for releases, indeed there is no indication on where to download releases.
But you can find pre-built binaries (although not signed) from the [releases](https://github.com/cvbarros/terraform-provider-teamcity/releases) page.

They are released automatically by CI on every tag.

**Comment by elucify** [comment]
Sorry for delayed response here.

Thanks, I didn't see the releases page before. (In fact now it seems obvious, I don't know how I missed it.)

As for errors, let me try again. Now that #62 is merged, maybe the build will work.

**Comment by elucify** [comment]
After checking out the master branch, `go get` produces:
```$ go get
# github.com/hashicorp/terraform/helper/plugin
../../hashicorp/terraform/helper/plugin/grpc_provisioner.go:173:6: b.Cap undefined (type strings.Builder has no field or method Cap)```

**Comment by elucify** [comment]
The Builder.Cap method was introduced in Go 1.12. So apparently the Go version 1.10 in the README needs to be updated to 1.12.*.

And indeed the build of `master` passes using 1.12.1.

**Comment by cvbarros** [comment]
Awesome, thanks for the input! I'll check it this weekend 👍

**Comment by augustgerro** [comment]
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/3669104/70226932-c5252080-1762-11ea-8a19-5a190e7e119b.png">
My Build issue on macOS

**Comment by augustgerro** [comment]
Probably, it's better provide completely finished binary?

**Comment by cvbarros** [comment]
You can find them on releases page:
https://github.com/cvbarros/terraform-provider-teamcity/releases
This was automated "recently" (basically the last work I did on the project), and docs are not up to date yet. Issue to tackle that: #58

**Comment by cvbarros** [comment]
> Sorry for delayed response here.
> 
> Thanks, I didn't see the releases page before. (In fact now it seems obvious, I don't know how I missed it.)
> 
You probably missed because the docs are misleading 😅  sorry about that, will put a fix over the weekend!

**Comment by cvbarros** [comment]
Instructions fixed via bd51bbb6c6b591a68d9c74bf1d1ac758694fb432
